### ISSUE TITLE
FR: Upgrade from OpenSSL 1.1

### ISSUE THREAD

**Issue by ekisu** [koreader/koreader#11686]
Right now, KOReader depends on OpenSSL 1.1 (at least, the `.deb` builds do), which is [EOL](https://endoflife.date/openssl). Are there any plans to migrate to a supported OpenSSL version?

**Comment by Frenzie** [comment]
The short answer is no, the longer answer is of course, but that's probably unlikely to happen within the next few months.

@benoit-pierre Do you happen to have any plans to touch that sooner?

**Comment by benoit-pierre** [comment]
No, I'd rather avoid conflicting work with the build system stuff.
